app/views/declaration/items_summary.scala.html

```diff
@@ -30,79 +42,130 @@
-                </tr>
-            </thead>
-            <tbody>
-
-            @items.zipWithIndex.map { case (item, index) =>
-                <tr id="item_@index">
-                    <th id="item_@index--sequence_id" scope="row">@item.sequenceId</th>
-                    <td id="item_@index--procedure_code">@item.procedureCodes.map(_.procedureCode).getOrElse("") </td>
-                    <td id="item_@index--item_type" >@item.commodityDetails.map(_.combinedNomenclatureCode) </td>
-                    <td id="item_@index--package_count">
-                        <ol type="1">@item.packageInformation.getOrElse(List.empty).map(packageInfo =>
-                            packageInfo.numberOfPackages.map(packages => <li>{packages}</li>)
-                        )
-                        </ol>
-                    </td>
-                    <td>
-                        <a id="item_@index--change" href="@routes.ProcedureCodesController.displayPage(mode, item.id)">
-                            <span aria-hidden="true">@messages("site.change")</span>
-                            <span class="visuallyhidden">@{messages("supplementary.itemsAdd.change.hint", item.sequenceId)}</span>
-                        </a>
-                    </td>
-                    <td>
-                        <a id="item_@index--remove" class="button button--gray" href="@routes.ItemsSummaryController.removeItem(mode, item.id)">
-                            <span aria-hidden="true">@messages("site.remove")</span>
-                            <span class="visuallyhidden">@{messages("supplementary.itemsAdd.remove.hint", item.sequenceId)}</span>
-                        </a>
-                    </td>
-                </tr>
-            }
+@changeLink(item: ExportItem) = {
+    <a class="govuk-link" href="@routes.ProcedureCodesController.displayPage(mode, item.id).url">
+        @messages("site.change")
+        <span class="govuk-visually-hidden">
+            @messages("supplementary.itemsAdd.change.hint", item.sequenceId)
+        </span>
+    </a>
+}
 
-            </tbody>
-        </table>
+@removeButton(item: ExportItem) = @{
+    govukButton(Button(
+        classes = "govuk-button--secondary govuk-!-margin-0",
+        content = Text(messages("site.remove")),
+        name = Some(messages("supplementary.itemsAdd.remove.hint", item.sequenceId)),
+        href = Some(routes.ItemsSummaryController.removeItem(mode, item.id).url)
+    ))
+}
+
+@noOfPackagesSection(item: ExportItem) = {
+    @if(item.packageInformation.getOrElse(List.empty).size > 1) {
+    <ol>
+        @item.packageInformation.getOrElse(List.empty).map(
+            packageInfo => packageInfo.numberOfPackages.map(
+                packages => <li>{packages}</li>
+            )
+        )
+    </ol>
+    } else {
+        @item.packageInformation.getOrElse(List.empty).map(
+            packageInfo => packageInfo.numberOfPackages.getOrElse("")
+        )
```

Comment:
Probably will need to spend more time to prepare something better, but just ideas in comments:
1. I'll extract `item.packageInformation.getOrElse(List.empty)` to local variable. You have this in three places (three times unwrapping Option) or you can wrap this in map like:
```
@item.packageInformation.map { packageInformationList =>
   // Logic
}
```
2. FYI Size is not efficient way of checking if there is one element. This method need to iterate through the whole collection. I know that this approach is more readable. My propose: `list.drop(1).isEmpty`

Reply:
Proposal, didn't test it, so don't know if it's ready code, but present the idea

```
@item.packageInformation.map { packageInformationList =>
   if(packageInformationList.drop(1).isEmpty) {
      packageInformationList.map { packageInfo =>
        packageInfo.numberOfPackages.getOrElse("")
      }
   } else {
      packageInformationList.map { packageInfo =>
         packageInfo.numberOfPackages.map( packages => 
            <li>{packages}</li>
         )
      }
   }
}
```

Reply:
Thank you :)
Shameless copy and past and a little tweak and things are working!